website_sale_preset_carts/models/time_frame.py

```diff
@@ -147,6 +189,7 @@ def generate_sale_orders(self):
             customers = (
                 self.env['res.partner']
                     .search([
+                        ('subscription_id', '!=', False),
                         ('subscription_id', '=', frame.subscription_id.id)
```

Comment:
frame_subscription_id.id could not be False. and if frame_subscription_id is false frame_subscription_id.id will raise an error.

So not sure ('subscription_id', '!=', False) is necessary

Reply:
See #25